pjlib/src/pj/ssl_sock_ossl.c

```diff
@@ -57,6 +57,26 @@
 #include <openssl/opensslconf.h>
 #include <openssl/opensslv.h>
 
+/* A single instance of server SSL context. */
+static SSL_CTX *server_ctx = NULL;
```

Comment:
instead of static var or singleton, is it possible to move this to ssl sock structure (e.g: only set for the listener socket)? so server ssl socks from the same listener will share the same SSL_CTX.

Reply:
okay. will try to incorporate Peter's patch as well.